Test Automation Course/src/inheritance/Masina.java

```diff
@@ -0,0 +1,28 @@
+package inheritance;
+
+public class Masina extends Vehicul {
+
+    private String marca;
+
+    public Masina(String marca, String culoare, int nrRoti) {
```

Comment:
rog sa adaugi chemarea constructorului clasei parinte prin intermediul cuvintului cheie super

Reply:
Am facut modificari. Le vei putea vedea in commitul care il voi face acum.